# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse_capi/examples/dump.rs
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd third_party/rust/mp4parse_capi/examples/dump.rs
--- a/third_party/rust/mp4parse_capi/examples/dump.rs	Fri Jan 27 20:35:34 2023 +0000
+++ b/third_party/rust/mp4parse_capi/examples/dump.rs	Fri Jan 27 22:54:08 2023 +0200
@@ -1,4 +1,11 @@
-use log::info;
+extern crate mp4parse;
+extern crate mp4parse_capi;
+
+#[macro_use]
+extern crate log;
+
+extern crate env_logger;
+
 use mp4parse::ParseStrictness;
 use mp4parse_capi::*;
 use std::env;
@@ -112,9 +119,7 @@
                         }
                     }
                 }
-                Mp4parseTrackType::Video
-                | Mp4parseTrackType::Picture
-                | Mp4parseTrackType::AuxiliaryVideo => {
+                Mp4parseTrackType::Video => {
                     let mut video_info = Mp4parseTrackVideoInfo::default();
                     match mp4parse_get_track_video_info(parser, i, &mut video_info) {
                         Mp4parseStatus::Ok => {